src/vs/workbench/contrib/files/browser/media/explorerviewlet.css

```
@@ -63,7 +63,8 @@
 }
 
 .explorer-folders-view .monaco-list-row[aria-expanded="false"] .explorer-item.highlight-badge .monaco-count-badge {
-	display: inline-block;
+	display: inline-flex;
+	align-items: center;
```

Testing locally, I noticed the text was pushed to the left a little bit. Adding `justify-content: center;` fixed that.

```suggestion
	align-items: center;
	justify-content: center;
```